src/commons/big_query/big_query_job_error.py

```diff
@@ -32,10 +34,13 @@ def should_be_retried(self):
         else:
             raise self.bq_error
 
+    def is_deadline_exceeded(self):
+        return self.bq_error.resp.status == 500 and \
+               self.full_reason.find('Deadline exceeded') != -1
```

Comment:
`and 'Deadline exceeded' in self.full_reason`

Reply:
Done :)